guides/common/modules/configuring_satellite_external_idm_dns.adoc

```diff
@@ -2,6 +2,10 @@
 
 = Configuring {Project} or {SmartProxy} with External IdM DNS
 
+ifeval::["{Build}" == "foreman"]
+This applies only to Red Hat-based systems.
+endif::[]
```

Comment:
I have noticed we use `Build` and `build` in our guides, it's likely that this is not case sensitive. I haven't checked, but I actually like `build` more as it comply with other variables. Maybe we could unify this sometimes (not is this PR please).

Reply:
@lzap I am addressing this in the Debian PR #100 !! Stay tuned!